### PR TITLE
ICU-20675 ignore .vscode configuration files.

### PR DESCRIPTION
**Pull request by younies**
##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/ICU-20675
- [x] Updated PR title and link in previous line to include Issue number
- [x] Issue accepted
- [ ] Tests included
- [ ] Documentation is changed or added

### CODE REVIEW

**Comment by sffc** [comment]
Please fill in the checklist in the OP, including the blank where you put the issue number.

File: .gitignore [hunk 1]
```diff
@@ -16,6 +16,7 @@ icu4c/lib32uwp
 icu4c/lib64
 icu4c/lib64uwp
 icu4c/libARMuwp
+icu4c/source/.vscode/*
```
Comment (sffc) [inline .gitignore]: You don't need the `/*` in .gitignore syntax.  If you give a directory path, the whole directory is ignored.

Comment (sffc) [reply .gitignore]: Actually, a better solution is to just put `.vscode/` at the top level.  That ignores all `.vscode` directories anywhere in the directory tree.

Comment (younies) [reply .gitignore]: Done :)

Thanks for the note.

**Comment by younies** [comment]
> Thanks, though I think this should likely be for the top level folder path and not inside the ICU4c subfolder.

Shane suggested this too, and I make it in the top.

**Comment by sffc** [comment]
Alternatively, my setup actually is that I have `.vscode/` as a global ignore in my personal git config settings.  So it's not strictly necessary to add it here.  But it does still reduce a step if you don't have your own personal gitignore file, so no harm done.

**Pull request merged**